server/src/controllers/poaching-incident-type.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Get, Query } from '@nestjs/common';
+import { PoachingIncidentTypeService } from '../services/poaching-incident-type.service';
+
+@Controller()
+export class PoachingIncidentTypeController {
+  constructor(
+    private readonly poachingIncidentTypeService: PoachingIncidentTypeService,
+  ) {}
+
+  @Get('addPoachingIncidentType')
+  async addPoachingIncidentType(
```

Comment:
Async not needed here - or can add "return await" on line 14

Reply:
Done